### PR TITLE
Generic 'Flippy' Tiles

### PR DESCRIPTION
**Pull request by sdougbrown**
We're making it easier to use the flipping tile styling!  This is a test case, converting just two spices to the new internal templates & variants.

Obviously, this requires an internal PR. :smile: 

to @bsstoner 
cc @moollaza @jagtalon

### CODE REVIEW

File: share/spice/get_events/get_events.js [hunk 1]
```diff
@@ -18,7 +18,8 @@
                     sourceName: "GetEvents",
                     sourceUrl: 'http://getevents.co',
                     itemType: "Upcoming Local Events",
-                    formattedSourceName: 'More on GetEvents'
+                    formattedSourceName: 'More on GetEvents',
```
Comment (bsstoner) [inline share/spice/get_events/get_events.js]: While we're here should we change this "More **on**" to "More **at**"? Everything else on UI says "More at".

**Comment by bsstoner** [comment]
:+1: waiting on internal changes

**Comment by bsstoner** [comment]
:+1:

File: share/spice/get_events/get_events.js [hunk 2]
```diff
@@ -28,21 +29,43 @@
...
                         city: item.venue.city,
-                        image: item.image_large_url,
-                        is_full_day: checkFullDay(item.start_date,item.end_date), 
-                        start_end: getStartEnd(item.start_date,item.end_date),
+                        place: item.venue.name,
                         lat: item.venue.lat,
                         lon: item.venue.lng
                     };
                 },
                 templates: {
```
Comment (talsraviv) [inline share/spice/get_events/get_events.js]: @bsstoner @moollaza how does this work without specifying sub-templates for front_content and back_content?

Comment (moollaza) [inline share/spice/get_events/get_events.js]: @talsraviv the `footer_content` for the front and back tiles are specified in `data_back` and `data_front`

- https://github.com/duckduckgo/zeroclickinfo-spice/pull/1981/files#diff-d135d1af653ff38be002528d7ef7af26R39
- https://github.com/duckduckgo/zeroclickinfo-spice/pull/1981/files#diff-d135d1af653ff38be002528d7ef7af26R50

**Comment by talsraviv** [comment]
Hmm - but `footer_content` isn't a feature of the `basic_flipping_item` template internally - what am I missing?

**Comment by talsraviv** [comment]
I documented it here - maybe easier to spot what I missed conceptually:
https://github.com/duckduckgo/duckduckgo-documentation/pull/296

**Pull request merged**